### PR TITLE
Jdp221201 13

### PR DESCRIPTION
**Pull request by MPawlow15**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -21,6 +21,8 @@ dependencies {
     annotationProcessor 'org.projectlombok:lombok'
     runtimeOnly('com.h2database:h2')
     testImplementation('org.springframework.boot:spring-boot-starter-test')
+    runtimeOnly 'mysql:mysql-connector-java'
```
Comment (interceptor2048) [inline build.gradle]: Test discussion

Comment (interceptor2048) [reply build.gradle]: Resolution

**Review by interceptor2048 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (interceptor2048) [reply build.gradle]: We agreed that it should look exactly as it looks

**Review by interceptor2048 — DISMISSED** [review]

**Review by julizale — APPROVED** [review]
Jest ... git 🤣

**Review by MichalRys90 — APPROVED** [review]

**Pull request merged**